pinot-common/src/main/java/org/apache/pinot/common/request/context/FunctionContext.java

```diff
@@ -84,7 +84,7 @@ public boolean equals(Object o) {
 
   @Override
   public int hashCode() {
-    return Objects.hash(_type, _functionName, _arguments);
+    return 31 * 31 * _type.hashCode() + 31 * _functionName.hashCode() + _arguments.hashCode();
```

Comment:
Do we need to see if arguments.hashCode() internally uses Objects.hash() and if that needs to be fixed too ?